src/sync/installation.ts

```diff
@@ -429,7 +431,7 @@ export const processInstallation = () => {
 			const nextTaskDelaysMs: Array<number> = [];
 
 			const result = await deduplicator.executeWithDeduplication(
-				"i-" + installationId + "-" + jiraHost,
+				`i-${installationId}-${jiraHost}-ghaid-${gitHubAppId}`,
```

Comment:
Instead of showing up `undefined`, can we add a condition for cloud and server, where the previous value is set for cloud and this new value is set for server?

Reply:
I agree. I don't think we should just have undefined here.

Reply:
I am fine as set for `cloud`, but it won't work to set for `server`, because this id is used to do the deduplication I believe. If two servers happen to have same `installationId`, then it will show `i-${installationId}-${jiraHost}-ghaid-server` and conflict.

Tbh as the id is only used in the key inside redis, so `undefined` technically works, but agree it doesn't looks nice. I can change that.

Reply:
fixed.